apps/ensadmin/src/components/recent-registrations/components.tsx

```diff
@@ -1,41 +1,68 @@
 "use client";
 
+import { ErrorInfo, ErrorInfoProps } from "@/components/error-info";
+import { RegistrationCard } from "@/components/recent-registrations/registration-card";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { useRawConnectionUrlParam } from "@/hooks/use-connection-url-param";
+import { cn } from "@/lib/utils";
 import type { ENSNamespaceId } from "@ensnode/datasources";
 import {
-  type ENSIndexerOverallIndexingCompletedStatus,
-  type ENSIndexerOverallIndexingFollowingStatus,
+  ENSIndexerOverallIndexingStatus,
   type ENSIndexerPublicConfig,
+  OverallIndexingStatusId,
   OverallIndexingStatusIds,
 } from "@ensnode/ensnode-sdk";
-import { fromUnixTime } from "date-fns";
+import { useAutoAnimate } from "@formkit/auto-animate/react";
+import Link from "next/link";
 import { useEffect, useState } from "react";
-
-import { Duration, RelativeTime } from "@/components/datetime-utils";
-import { NameDisplay, NameLink } from "@/components/identity/utils";
-import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
-import {
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableHeader,
-  TableRow,
-} from "@/components/ui/table";
-import { Identity } from "../identity";
 import { useRecentRegistrations } from "./hooks";
-import type { Registration } from "./types";
 
 /**
  * Max number of latest registrations to display
  */
-const MAX_NUMBER_OF_LATEST_REGISTRATIONS = 5;
+const DEFAULT_MAX_ROWS = 25;
 
-interface RecentRegistrationsProps {
-  ensIndexerConfig: ENSIndexerPublicConfig;
+/**
+ * Omnichain indexing statuses that allow the display of registrations.
```

Comment:
```suggestion
 * Omnichain indexing statuses where ENSAdmin allows itself to query registrations.
```

Note: We need to separate the idea of what is "allowed" technically (by ENSNode overall) vs what we (ENSAdmin) allow ourselves to do.

Technically it's allowed (by ENSNode overall) to always query registrations. But we don't allow ourselves to do it unless the overall omnichain indexing status is one we (ENSAdmin) decides to make a query for.